From 3ff674035d8962bf3ca3320900fccc6619af7a3d Mon Sep 17 00:00:00 2001
From: Adam Stokes <adam.stokes@ubuntu.com>
Date: Fri, 14 Mar 2014 15:03:29 -0400
Subject: [PATCH 36/72] Do not collect isos in cobbler plugin

In Ubuntu isos are being added to the tarball, do not do that.

Signed-off-by: Adam Stokes <adam.stokes@ubuntu.com>
---
 sos/plugins/cobbler.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sos/plugins/cobbler.py b/sos/plugins/cobbler.py
index e3d289e..b754813 100644
--- a/sos/plugins/cobbler.py
+++ b/sos/plugins/cobbler.py
@@ -39,3 +39,4 @@ class DebianCobbler(Cobbler, DebianPlugin, UbuntuPlugin):
         self.add_copy_spec("/etc/cobbler")
         self.add_copy_spec("/var/log/cobbler")
         self.add_copy_spec("/var/lib/cobbler")
+        self.add_forbidden_path("/var/lib/cobbler/isos")
-- 
1.9.3

